packages/frontend/src/components/MenuBar/navigationDropdownMenu.module.css

```diff
@@ -21,22 +28,6 @@
   z-index: 1001;
 }
 
-.menuItem {
```

Comment:
Hvorfor fjerner du alt dette som jeg gjorde for å fikse designet i saved searches?